packages/checkbox-group/test/lion-checkbox-indeterminate.test.js

```diff
@@ -428,4 +429,80 @@ describe('<lion-checkbox-indeterminate>', () => {
     expect(elIndeterminate?.hasAttribute('indeterminate')).to.be.false;
     expect(elIndeterminate?.checked).to.be.true;
   });
+
+  // https://www.w3.org/TR/wai-aria-practices-1.1/examples/checkbox/checkbox-2/checkbox-2.html
+  describe('mixed-state', () => {
+  it('can have a mixed-state (using mixed-state attribute), none -> indeterminate -> all, cycling through', async () => {
+    it('can have a tri-state (using tri-state attribute), none -> indeterminate -> all, cycling through', async () => {
```

Comment:
Now there is a double "it"

```suggestion
```

Reply:
Giving me bad suggestions to make CI fail, I seeeee hehe